docs/howtos/telemetry/30-metrics-qs.md

```diff
@@ -31,7 +31,36 @@ that allows us to define Prometheus' Targets intuitively.
 There are many ways to install and set up Prometheus. For ease of deployment, we will use the
 Prometheus community Helm chart.
 
-Let's install the Prometheus stack Helm Chart:
+Let's create a `kube-prometheus-stack-values.yaml` file with the
+following contents:
+
+````yaml
+---
+prometheus:
```

Comment:
isn't this redundant with the two `ServiceMonitor` resources created below?

Reply:
Indeed, dropped them in https://github.com/kubewarden/docs/pull/431/commits/95ad718e5e33778e462ca1cfd5bcdc9eae8f1e66.